setup.py

```diff
@@ -142,7 +141,7 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
     long_description=LONG_DESCRIPTION,
     platforms=['any'],
     classifiers=CLASSIFIERS,
-    install_requires=["boto>=2.31.0", "pytz>=2016.10", "future>=0.16.0"],
+    install_requires=["boto3>=1.0.0", "pytz>=2016.10", "future>=0.16.0", "django>1.11"],
```

Comment:
```suggestion
    install_requires=["boto3>=1.0.0", "pytz>=2016.10", "future>=0.16.0"],
```

Reply:
Why did you remove the django dependency here?

Reply:
I removed it because because Travis matrix has Django 1.11 (https://github.com/django-ses/django-ses/blob/master/.travis.yml#L12) and because it was not on the initial source (https://github.com/django-ses/django-ses/blob/master/setup.py#L145), but maybe it's a good thing to add it?

Reply:
I mainly added it, because the IDE gives you loades of warnings if it's missing 😅